### ISSUE TITLE
Default CI level back to 95%? 😱

### ISSUE THREAD

**Issue by DominiqueMakowski** [easystats/bayestestR#250]
## TLDR for new readers

Currently, most of bayestestR's functions have `ci=0.89` by default, yielding 89% CIs. We are thinking of changing that (maybe to 95%). To avoid any surprises or breaks when you update the package, set the `ci` argument explicitly. For example, to retain the current behaviour, just add `ci=0.89` to most of bayestestR's functions (or `parameters::model_parameters()`).

## Discussion

Might be throwing a stone in the water here, but it's good to rechallenge our certainties :)

When we switched from 95% to 89%, the main reason was that there was no objective reason to keep 95% as it is a purely arbitrary value. And 89%, suggested by McElreath, has *at least* for it to be a prime number (not that it changes anything, but it might be nice [for some strange people such as @strengejacke 😁]). The real underlying motivation was also to shake the common (and mindless) procedures, to make people actually *think* about what they are doing. 

Now, I think I have some new arguments to throw in the pot.

- 89% is aesthetically ugly. Don't kill the messenger here, but it's true ([see this](https://psychology.stackexchange.com/questions/3514/aesthetic-preference-for-even-or-odd-numbers/5807), or [Eilers, 2004](https://www.ingentaconnect.com/content/iuatld/ijtld/2004/00000008/00000002/art00011)). We tend to prefer round values (and having an aesthetically pleasing world *is* very important). For regular folks as me, 89 has a particular taste of unfinished business. 
- 95% can facilitate bridges with the Frequentist framework. In times of reproducibility crisis, it doesn't help to change all the references. Keeping 95% would at least allow for some level of comparison with frequentist studies (as credible and confidence intervals are in the end often close), and would also facilitate replication attempts and checks. Note that the downside of this argument is that it creates a meta-stable system in which moving away from bad procedures is very complicated (because by following this argument, one could argue to keep using old methods for continuation and comparison purposes).
- 89 is too small? IMO one of the goals of credible intervals is to address the fact that many distributions have virtually no ends. For instance, in a normal distribution of mean 0 and SD 0.1, a value of 10^99 is still (although infinitesimally) possible. Thus, the idea was to take a range of the most plausible values, mainly for convenience, to say "this distribution goes from here to here". Hence the approximation we learn in class that a normal distribution has a range of 6 SD, because it makes some practical sense. Within this context, 89 appears as being a bit too narrow as a summary of an (at least normal) distribution, cutting the tails quite heavily.
- Relationship with SD. If we take our beloved normal distribution (which is the most "common" (or aspecific) distribution), 95% is actually an approximation in the so-called [68, 95, 99.7 rule](https://en.wikipedia.org/wiki/68%E2%80%9395%E2%80%9399.7_rule). 95% is the approximation of 0.9544997, which is the range of the population contained within 2 SDs on each side of the mean. Although this 95% value likely originated precisely from this relationship with the SD in the frequentist framework (allowing for some nice equation simplifications), the reason to use it in a Bayesian framework would be to connect the CI to the SD (instead of reporting both, which I never liked), which is also a familiar and graspable index.

All opinions are appreciated

```r
# Code for CDF
x <- bayestestR::distribution_normal(10^8)
length(x[x <= 2 & x >= -2]) / 10^8
0.9544997
```

**Comment by strengejacke** [comment]
But some of the [coolest papers](https://bmjopen.bmj.com/content/9/9/e030743.full) use 89% HDI... ;-)

> 89% is aesthetically ugly

![3e45zj](https://user-images.githubusercontent.com/26301769/67399441-8c632a80-f5ac-11e9-8903-3ffd74c72492.jpg)

**Comment by strengejacke** [comment]
Of course, there is no reason _against_ 95% (as there is also no reason _against_ 89%). Computational stability is an argument from the Stan developers (and Kruschke) to use 90%, so where to go now? Is 89 to narrow? is 95 to imprecise? is 90 ok (but if we break conventions, why not stick to 89?)?

I'm not dogmatic here and open for any changes. One reason _against_ 95% is, and I will cite (and translate) a German song text:

> We do not have to do things as we used to do, just because we used to do it as we used to do it.

(not sure if this translation really works...?)

**Comment by mattansb** [comment]
I thought some of the motivation was Kruschke's idea that for a 95% CI you need a lot of data, and so it would be no more than 90%?

**Comment by DominiqueMakowski** [comment]
"Computational stability" is indeed one of the strong argument for lowering the percentage. But I'd be curious to see some figures showing the impact of the CI level on the results... does any know where to find that?

**Comment by strengejacke** [comment]
>  does any know where to find that?

What about those books or papers where this claim stems from?

**Comment by strengejacke** [comment]
I don't understand everything, but I _think_ this post at least contributes a bit to your question...

https://betanalpha.github.io/assets/case_studies/markov_chain_monte_carlo.html

**Comment by strengejacke** [comment]
Discussion resolved?

**Comment by strengejacke** [comment]
close to being closed... ;-) 
(_bump_)

**Comment by DominiqueMakowski** [comment]
I think I have become pro 95 tho :sweat_smile: but yeah I guess we've gone too far with 89 to make such a breaking change

**Comment by strengejacke** [comment]
Let's wait for the 2nd edition of _Statistical Rethinking_ (should be in my office right now...) and if there's still the 89%, we will close this, else we'll make a breaking change to 95.

**Comment by strengejacke** [comment]
Ok, in the current (2nd) edition of Statistical Rethinking, it seems like McElreath still advices _against 95%_ CI (just to avoid falling into the trap to think of NHST when you hear "95%"), but he doesn't seem to say 89% is _the_ number. So... We could

a) stick to 89%
b) change to 90% (like rstanarm)
c) change to 95% (like brms and frq)

should we run `runif()` to decide? I prefer a) or b).

**Comment by strengejacke** [comment]
_and the winner is..._

``` r
table(sapply(1:1000, function(i) sample(c("a", "b", "c"), 1)))
#> 
#>   a   b   c 
#> 343 331 326
```

<sup>Created on 2020-05-11 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by mattansb** [comment]
lol

I also prefer a or b (b a bit more).

**Comment by strengejacke** [comment]
I'm open to either a) or b) 
@DominiqueMakowski you're the maintainer...

**Comment by DominiqueMakowski** [comment]
the time that has passed has strongly reinforced my preference for *c* 😅 it seems like we are in a pickle...

The argument of not doing something just because it reminds of something not good is a bit weak imho. And *au contraire* you could argue that 95% is great *because* it allows for comparison with previous results and, you know, reproducibility and all that kind of jazz. But aside from that, the real game changer for me is the correspondence between the 95% CI and the SD (or the SE - right @strengejacke :p)...

**Comment by DominiqueMakowski** [comment]
If being maintainer doubles my vote weight, we're still stuck between 2c vs. 2b 😁 

also, I do remember you were more a brms than a rstanarm guy @strengejacke 🤔

**Comment by mattansb** [comment]
What is the correspondence between 95% CI and SD?

(And a reminder: the SD of the posterior is not the SE 😋)

**Comment by strengejacke** [comment]
> also, I do remember you were more a brms than a rstanarm guy

Yes, I like the flexibility of brms, but... that doesn't mean I would blindly follow all conventions from the brms-package, though ;-)

**Comment by strengejacke** [comment]
> What is the correspondence between 95% CI and SD?

I think he refers to

```r
x <- bayestestR::distribution_normal(10^8)
length(x[x <= 2 & x >= -2]) / 10^8
#> 0.9544997
```

**Comment by DominiqueMakowski** [comment]
That's it, I'm writing a blogpost to present my point and open the debate. I can point to that issue so people can give their opinion? we might have over/underlooked some elements :)

**Comment by strengejacke** [comment]
> we might have over/underlooked some elements :)

Maybe you did, I did not. 🚀

😄

**Comment by strengejacke** [comment]
> What is the correspondence between 95% CI and SD?

I don't think @mattansb sees the same strong correspondence as you do: https://github.com/easystats/parameters/issues/256 ;-)

**Comment by DominiqueMakowski** [comment]
most of the model posteriors are akin to normal distributions (as they are usually specified as priors 🤷). And still, while this correspondence it's not a mathematical truth and many counter-examples could be found, it could be seen as a heuristic, giving some intuition and linking in the minds of the young Jedi different concepts such as credible intervals and SDs...

**Comment by mattansb** [comment]
> `x <- bayestestR::distribution_normal(10^8)`
`length(x[x <= 2 & x >= -2]) / 10^8`
`#> 0.9544997`

Isn't this why Fisher set alpha at 5% in the first place? When M / SE<sub>M</sub> > 2 is significant?

But SD<sub>posterior</sub> is not an SE (the posterior is not the same as the sampling distribution, and also if you really think about it, in a Bayesian perspective there isn't *an* SE, but a whole distributions of SEs!)

I also recall reading that "good" posteriors are inherently not symmetrical (or normal)... But I can't seem to find that paper.

**Comment by mattansb** [comment]
Option d = remove the default completely! *Force* users to make their own damn minds!

**Comment by strengejacke** [comment]
> But SDposterior is not an SE

This is an old discussion between Dom and me, I think it was how we name the SD / SE from Bayesian R2, which currently prints ["Bayesian R2 with Standard Error"](https://easystats.github.io/performance/reference/r2_bayes.html). And on Twitter msg I posted [this link](https://statmodeling.stat.columbia.edu/2020/05/09/standard-deviation-standard-error-whatever/#comment-1333367). 

It's just confusing that the SD ~ SE when comparing models:

``` r
library(rstanarm)
model <- stan_glm(mpg ~ wt + gear, data = mtcars, chains = 2, iter = 200, refresh = 0)
summary(model)
#> 
#> Model Info:
#>  function:     stan_glm
#>  family:       gaussian [identity]
#>  formula:      mpg ~ wt + gear
#>  algorithm:    sampling
#>  sample:       200 (posterior sample size)
#>  priors:       see help('prior_summary')
#>  observations: 32
#>  predictors:   3
#> 
#> Estimates:
#>               mean   sd   10%   50%   90%
#> (Intercept) 39.0    5.3 31.6  38.7  46.0 
#> wt          -5.5    0.7 -6.4  -5.6  -4.6 
#> gear        -0.3    0.9 -1.5  -0.3   1.0 
#> sigma        3.1    0.4  2.7   3.1   3.6 

model2 <- lm(mpg ~ wt + gear, data = mtcars)
summary(model2)
#> 
#> Call:
#> lm(formula = mpg ~ wt + gear, data = mtcars)
#> 
#> Residuals:
#>     Min      1Q  Median      3Q     Max 
#> -4.1304 -2.3061 -0.2932  1.4409  6.8296 
#> 
#> Coefficients:
#>             Estimate Std. Error t value Pr(>|t|)    
#> (Intercept)  38.9157     5.0974   7.634 2.04e-08 ***
#> wt           -5.4850     0.6987  -7.851 1.17e-08 ***
#> gear         -0.3196     0.9265  -0.345    0.733    
#> ---
#> Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1
#> 
#> Residual standard error: 3.092 on 29 degrees of freedom
#> Multiple R-squared:  0.7538, Adjusted R-squared:  0.7369 
#> F-statistic: 44.41 on 2 and 29 DF,  p-value: 1.488e-09
```

<sup>Created on 2020-05-11 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

_(though **parameters** does it "right"...)_

``` r
model_parameters(model, dispersion = TRUE, centrality = "mean")
#> # Fixed effects 
#> 
#> Parameter   |  Mean |   SD |         89% CI |     pd | % in ROPE |  Rhat | ESS |               Prior
#> ----------------------------------------------------------------------------------------------------
#> (Intercept) | 38.93 | 5.57 | [29.69, 46.72] |   100% |        0% | 0.991 | 202 | Normal (0 +- 60.27)
#> wt          | -5.50 | 0.76 | [-6.79, -4.39] |   100% |        0% | 0.992 | 213 | Normal (0 +- 15.40)
#> gear        | -0.28 | 0.99 | [-1.61,  1.31] | 66.50% |    48.50% | 0.994 | 205 | Normal (0 +- 20.42)
```

<sup>Created on 2020-05-11 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by mattansb** [comment]
SD<sub>posterior</sub> is ~ to SE<sub>H0</sub> with relatively flat priors, for *some* parameters, in simple models... This is because in such cases the posterior is basically the likelihood function, and the MLE is approximately the same as the OLS estimate etc... But for any other case, the numerical similarities don't hold as much if at all...

But numbers are just numbers - the real problem is with meaning, and the meaning of SD<sub>posterior</sub> and of SE<sub>H0</sub> are completely different (just like `pd` and p-value are different in meaning even though they often are similar numerically)!

**Comment by mattansb** [comment]
But as you said, we've had this discussion before 🤪
(But that won't stop me from stating my case every-single-time 😇😅)

**Comment by strengejacke** [comment]
But in essence, this means there is no "real" correspondence between 95% CI and SD, and therefore 

> the real game changer for me is the correspondence between the 95% CI and the SD

is actually no gamechanger, but a "nonstarter". ;-)

**Comment by mattansb** [comment]
![image](https://user-images.githubusercontent.com/35330040/81613069-7a30c880-93e6-11ea-9d9e-27b2a87299f3.png)

The logic for why Credible interval are not always equal to confidence interval,
can be translated to why pd is not p-value, and why SD<sub>posterior</sub> is not SE (:

FIN

**Comment by mattansb** [comment]
How about `ci(x, ci = runif(1,0,1))`? 🤪

**Comment by strengejacke** [comment]
Fixed:
`ci(x, ci = runif(1,0,.94))`

**Comment by dmenne** [comment]
Speaking as  someone working in clinical studies: It is a dreadfult uphill battle to argue somewhat along the "think about p-values" line in Amrhein et al. The answer is an automatic: "but the reviewer wants p-values". Where "reviewer" means some big-shot medical professor who argues along the lines "I have use Wilcoxon all my life, it solves all". 

Check the journal guidelines, and they are much less p-fixed than the supposed reviewer, but nobody reads guidelines.

Why this here? Because it is so difficult to plea for confidence or credible intervals in our job. Against my will I often argue "If zero it is outside of the CI, it is significant (in you old terms)", but  please don't ask me to open a second front arguing against 95%. 

McElreath has many merits, but the 89% was not his brightest idea.

**Comment by ingorohlfing** [comment]
On the one hand, McElreath has a point and proposing 0.89 underscores it. On the other hand, exactly because any CI width is arbitrary, nothing speaks for 0.89 over 0.95 and the other way round (prime vs not prime is not a strong reason). Ideally, a researcher thinks about the proper CI before the analysis is done and chooses the preferred width (which probably happens less often than it should be). In practice, I would prefer consistency of default settings across package versions, mainly for reproducibility reasons (if renv() or so is not used). I therefore suggest keeping it 0.89.

**Comment by dmenne** [comment]
You think "a researcher thinks about the proper CI"?  No, the researcher wants comparability with other people's results, and not fight in the wrong trench just to show this free will.

**Comment by mattansb** [comment]
But should R packages' defaults cater to such behaviors or situations?

**Comment by dmenne** [comment]
The posters here are of the statisticians group, while easystat tries to speak to researchers using statistics as they use their Pipetman or Oscilloscope. Reach out to users, not to developers.

I don't care very much what you use, because I do not use easystat, but you will loose your customer base when you try to push your own story - nobody but Elreath uses 0.89. Provocation alone must have a worthy cause, fighting against the misuse of p-values is worth it; thresholds are not.

**Comment by ppernot** [comment]
Many scientific communities recommend to publish results with enlarged uncertainty
aiming at 95% coverage intervals. 
For instance in chemistry: [B. Ruscic. Int. J. Quantum Chem. 2014,
114, 1097–1101.]( https://doi.org/10.1002/qua.24605)

**Comment by DominiqueMakowski** [comment]
Another argument in favour of 95 that should appeal to @strengejacke pertains to its use to assess significance:

A long time ago we were discussing something like ROPE or equi tests and you were particularly keen on choosing one option because it led to more conservative results (i.e., harder to reject H0), which was in your opinion a good thing if I remember correctly. 

Well, in that case, if one is using the CI to assess significance (for instance whether it covers 0 or if it falls in the ROPE), 95% CI also leads to more conservative results than 89% ☺️

**Comment by DominiqueMakowski** [comment]
> In practice, I would prefer consistency of default settings across package versions, mainly for reproducibility reasons (if renv() or so is not used). I therefore suggest keeping it 0.89.

Keeping consistency is a major argument indeed, and this would be a breaking change for many users indeed. 

However, I don't think it's a blocker for several reasons. The first is conceptual; in general I don't think that package developers should stick eternally with bad decisions (or refrain for updating to better decisions) just because of stability, a recent example being the infamous `stringsAsFactors=TRUE` default in R that is finally being changed in the new version. Moreover, technically bayestestR is a still a young package in its 0.x version, which in theory means that breaking changes like this can happen.

That being said, it remains a valid concern, and we need to think of ways to mitigate such breaking changes (whether we do it for this or something else in the future). The smoothest way is to first replace the default argument (let's say `ci=0.89`) by something like `ci='default'`. Then, for a couple of releases, we still replace 'default' by `0.89` but print a message saying that this default behaviour will be updated in the near future, so "please specify this value explicitly". Then, after a reasonable amount of time, we make the change along with some advertisement. Hopefully this way, we will keep the problems to a minimum

**Comment by strengejacke** [comment]
> Another argument in favour of 95 that should appeal to @strengejacke pertains to the use of the CI to assess significance:
> 
> A long time ago we were discussing something like ROPE or equi tests and you were particularly keen on choosing one option because it led to more conservative results (i.e., harder to reject H0), which was in your opinion a good thing if I remember correctly.
> 
> Well, in that case, if one is using the CI to assess significance (for instance whether it covers 0 or if it falls in the ROPE), 95% CI also leads to more conservative results than 89% ☺️

I think inspecting uncertainty in general and judging the size / magnitude of an effect are too different things, thus I don't see to have same numbers here.

This is what McElreath says (writes) now:

![IMG_20200515_072446](https://user-images.githubusercontent.com/26301769/82015275-afefde80-967e-11ea-9c63-9988fb515b68.jpg)

To summarize, using 95% would mean:
- you consider "convention" as a serious statistical criterion
- you don't like thinking
- you like conducting unconsciousness hypothesis-testing where it is not valid

🚀

**Comment by DominiqueMakowski** [comment]
> I think inspecting uncertainty in general and judging the size / magnitude of an effect are two different things

While I may agree in theory, yet the CI range set by default is used in both cases, in `ci()` and for instance in `equivalence_test()` 😏

**Comment by strengejacke** [comment]
I think the best "ci" we can find in Bayesian statistics should probably benefit from the fact that we can do _probability statements_ about an effect. That is actually something we have with [`p_significance()`](https://easystats.github.io/see/articles/bayestestR.html#practical-significance-1). However, this is of course no static value...

I see the points to vote for 95%, I still think .89 and .9 is better, but I'm would not persist on this limit and am OK with 95% as well. In the end, it's a changeable default. I think, in the end, most users probably care less about the default than we do here, so it's no big deal anyway. But what I liked about our default is indeed that it encouraged "statistical rethinking". :-)

**Comment by strengejacke** [comment]
https://twitter.com/polesasunder/status/1264527314676707328?s=19

**Comment by strengejacke** [comment]
https://twitter.com/rlmcelreath/status/1288761673432600577?s=19

**Comment by mattansb** [comment]
🤣🤣🤣👏👏👏

**Comment by jacopastorius** [comment]
As your default, suggest:

(1) not reporting any point estimate of a central tendency (median, mean or mode).
(2)report a 50% _Uncertainty Interval_  for the location and use that term (https://statmodeling.stat.columbia.edu/2009/05/24/handy_statistic/)
(3) report a continuous PD statistic for the uncertainty bound. 

This would cultivate a better habit of thinking about uncertainty during inference, and steer people away from the all-or-nothing- thresholds.

(2)
Also, it'd be nice to have reports exportable as tables to HTML and LaTeX (which would be very helpful for scientific publishing).

(3) 
Very grateful for your work. You really make my life so much easier, and its such a joy to work with your tools. Thank you.

**Comment by DominiqueMakowski** [comment]
Thanks @jacopastorius for chiming in the conversation ☺️ 

So if I understand correctly, you suggest swapping the centrality and "credible interval" indices by only one narrower (e.g., 50%) *uncertainty* interval. 

Several things come to my mind:

1) Though I understand the purpose of the semantic change to *uncertainty* interval, especially from *confidence* interval (as it underlines the uncertainty rather than reinforcing the illusion of certainty), I think that *credible* interval is okay too, as it refers to an "objective" aspect of the distribution (a proportion of the credible mass of its density).

2) dropping the index of centrality seems a bit extreme to me. Aside from the fact that they are used a lot (sometimes meaningfully, for instance, variational Bayes is a whole framework based (*bayes'd*) on them), some of them make a lot of sense from a probabilistic perspective. Especially the median, which beyond its numerical stability has a very clear meaning: there is 50% of chance that the coefficient is larger and 50% that it is lower.

3) from 2, dropping such "objective" description of the posterior in favour of some arbitrary range (why 50% and not 42%?) seems a bit anti-*"cultivate a better habit of thining about stuff"* mindset 

> Also, it'd be nice to have reports exportable as tables to HTML and LaTeX (which would be very helpful for scientific publishing).

hopefully, in a not too distant future with [**report**](https://github.com/easystats/report) 😁

**Comment by jacopastorius** [comment]
Good thoughts @DominiqueMakowski.  You're right, too extreme. Though perhaps not as extreme as I sounded. I was merely speaking package default setting.  It's really important to be able to recover indices of centrality and uncertainty.  People benefit from the flexible toolkit you've developed ... Thanks again for all your work. I sincerely appreciate what you and your collaborators are doing here.

**Comment by DominiqueMakowski** [comment]
@strengejacke @mattansb soooo did your opinions about this change after summer? I'd say we should go ahead with this. Maybe we can start by adding a startup message saying that it will change in the future version?

Another point that just crossed my mind is that McElreath's argument for 89% was made precisely against any "standard", and to say that basically any value is acceptable and none is more legit than another. But it was not made to create a new standard...

![](https://i.redd.it/9o0obhrr92u01.jpg)

So I'd say we set the default to 95%, as this is a default by all historical and mathematical reasons, *and* we explain in the CI vignette (and in the docstrings) what we already explain, i.e., that other values are just as good if not better.

**Comment by mattansb** [comment]
I am (remain?) unopinionated...

If we do go ahead with this change, a nice big startup message is a must.

> Another point that just crossed my mind is that McElreath's argument for 89% was made precisely against any "standard"

So this means that every year or so we should switch between 0.89 and 0.95?

**Comment by strengejacke** [comment]
> So this means that every year or so we should switch between 0.89 and 0.95?

😄 

we should default `ci = runif(1)`...

**Comment by strengejacke** [comment]
> Maybe we can start by adding a startup message saying that it will change in the future version?

Maybe we can start by adding a startup message saying that it **may** change in the future version?

**Comment by mattansb** [comment]
👆 definitely this!
"Default CI width may change in the future, so it's safest to specify them, even if equivalent to the defaults."

**Comment by DominiqueMakowski** [comment]
`The default CI width (currently 89%) might change in future versions (see https://github.com/easystats/bayestestR/issues/250). To avoid any issues, please set the value explicitly by adding 'ci=0.89' to your bayestestR functions.`

**Comment by mattansb** [comment]
But here you are instructing users what they should set the value to - but we want to encourage them to _think for their own_ what the CI should be for their domain / use case. This isn't just about avoiding any issues, it is about avoiding defaults here as a general guideline.

So:
```
The default CI width might change in future versions (see https://github.com/easystats/bayestestR/issues/250). 
To prevent any issues, please set them explicitly when calling bayestestR functions.
```

**Comment by strengejacke** [comment]
we may tag Richard McElreath, just to get another unbiased opinion here...? 😇

**Comment by DominiqueMakowski** [comment]
true, but with some concrete example it's even better 😬  

```
The default CI width might change in future versions (see https://github.com/easystats/bayestestR/issues/250). 
To prevent any issues, please set it explicitly when calling bayestestR functions (e.g., 'ci=0.89' to retain the current behaviour).
```

**Comment by DominiqueMakowski** [comment]
> we may tag Richard McElreath, just to get another unbiased opinion here...? 😇

tho I reckon it might be a slightly biased opinion 😅

**Comment by mattansb** [comment]
> e.g., 'ci=0.89' to retain the current behaviour

Not a fan - we're cementing 0.89 here again...
How about:

```
The default CI width might change in future versions (see https://github.com/easystats/bayestestR/issues/250). 
To prevent any issues, please set it explicitly when calling bayestestR functions, via the 'ci' argument.
```
